src/index.js

```diff
@@ -0,0 +1,181 @@
+import './style.css';
+import { countElements } from './comments.js';
+
+const getLink = 'https://api.nasa.gov/planetary/apod?api_key=4SuCuNxM0J6w1FmjQokTcawsubomH7aV4ep60VgT&start_date=2021-03-16&end_date=2021-03-21';
+const datesForPopup = ['2021-03-16', '2021-03-17', '2021-03-18', '2021-03-19', '2021-03-20', '2021-03-21'];
+const getImage = (num) => `https://api.nasa.gov/planetary/apod?api_key=4SuCuNxM0J6w1FmjQokTcawsubomH7aV4ep60VgT&date=${datesForPopup[num]}`;
+const main = document.getElementById('addToScreen');
+const starLink = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/9mAPgvMc6PjOJk4JU1ZU/likes';
+const commentLink = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/9mAPgvMc6PjOJk4JU1ZU/comments';
+
+const getScores = async (url) => {
+  const response = await fetch(url);
+  return response.json();
+};
+
+const postScores = async (url, data) => {
+  const response = await fetch(url, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(data),
+  });
+  return response;
+};
```

Comment:
- [ ] Please handle fetch request failures.

Reply:
Could you clarify this? which request failures? Maybe error screenshot can be helpful

Reply:
- [ ] The issue here has not been properly resolved. To clarify this, what the reviewer means is to write code that'll manage the fetch request failures. e.g If there's no data to fetch what should happen, check this [link](https://www.codegrepper.com/code-examples/javascript/fetch+api+async+await+try+catch) for an example